yarn-project/circuits.js/src/structs/call_context.ts

```diff
@@ -94,6 +112,7 @@ export class CallContext {
       new AztecAddress(reader.readBytes(32)),
       new AztecAddress(reader.readBytes(32)),
       new EthAddress(reader.readBytes(32)),
+      FunctionSelector.fromBuffer(reader.readBytes(4)),
```

Comment:
assuming we don't have a constant variable anywhere that denotes how large, in bytes, the selector is?

Reply:
Hmm, think there actually is.